### PR TITLE
docs: Minor tweaks (README, bib ref, notebook instructions, some directives)

### PR DESCRIPTION
**Pull request by EricCousineau-TRI**
This is just a grab-bag of minor edits. When filing #2533, had seen a minor issue in the docs. Came back to try and fix, didn't see it, but saws these minor things.

Perhaps the most controversial is the image linebreak; in my browser (Firefox 82.0, Ubuntu 18.04), it looks a tad odd in Sphinx due to the `mdinclude` directive (no horizontal space btw img and text)
image

---
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/2534)

### CODE REVIEW

**Comment by EricCousineau-TRI** [comment]
Will avoid updating CHANGELOG for now, as these changes seem trivial. Can do so if a person asks!

Also, lemme know if it's worth anyone's time for me to break this up.

File: README.md [hunk 1]
```diff
@@ -72,19 +72,19 @@ and [Open3D C++ interface](http://www.open3d.org/docs/release/cpp_api.html).
 
 ## Open3D-Viewer app
 
-<img align="left" width="480" src="https://raw.githubusercontent.com/intel-isl/Open3D/master/docs/_static/open3d_viewer.png">
+<img width="480" src="https://raw.githubusercontent.com/intel-isl/Open3D/master/docs/_static/open3d_viewer.png"/>
 
 Open3D-Viewer is a standalone 3D viewer app available on Ubuntu and macOS.
 Please stay tuned for Windows.
 
 Download Open3D Viewer from the
```
Comment (yxlao) [inline README.md:80]: I would merge this with the previous paragraph. Now if we are not doing the left-right alignment, one sentence per paragraph seems a bit sparse.

File: README.md [hunk 2]
```diff
@@ -72,19 +72,19 @@ and [Open3D C++ interface](http://www.open3d.org/docs/release/cpp_api.html).
...
 [release page](https://github.com/intel-isl/Open3D/releases).
 
-<br clear="left"/>
+<br/>
```
Comment (yxlao) [inline README.md:83]: remove this `<br/>`

File: README.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ top of the Open3D core library and extends it with machine learning tools for
 To try it out, install Open3D with PyTorch or TensorFlow and check out
```
Comment (yxlao) [inline README.md:93]: Same, I would merge this with the previous paragraph.

File: README.md [hunk 4]
```diff
@@ -93,7 +93,7 @@ top of the Open3D core library and extends it with machine learning tools for
...
 [Open3D-ML](https://github.com/intel-isl/Open3D-ML).
 
-<br clear="left"/>
+<br/>
```
Comment (yxlao) [inline README.md:96]: remove this `<br/>`

**Comment by yxlao** [comment]
hi @EricCousineau-TRI i've adopted some of your changes in `README.md` to remove the left-right alignment. You may need to merge `README.md` from `master` again,

**Review by yxlao — APPROVED** [review]

**Pull request merged**